### PR TITLE
Falcon fix

### PR DESCRIPTION
**Pull request by Soundes-M**
I couldn't have the pqm4 benchmark data

### CODE REVIEW

File: signatures/FALCON/FALCON.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - Thomas Ricosset
+  - Gregor Seiler
+  - William Whyte
+  - Zhenfei Zhang
+problems:
+    - assumption: SIS over NTRU lattices
+stateful: false
+nist round: 2
+sources:
+  - NIST Submission Paper
```
[the remark this answers is not in the document]
Comment (Soundes-M) [reply signatures/FALCON/FALCON.yaml]: yes

File: signatures/FALCON/FALCON/bench/avx2_512_Skylake.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+platform: Intel Core i7-6567 (Skylake) 3.3 GHZ
+timings:
+  unit: cycles 
+  gen:  26604000
+  enc|sign: 531684
```
Comment (dingens) [inline signatures/FALCON/FALCON/bench/avx2_512_Skylake.yaml:5]: The timings used are sometimes from the `(tree, ct)` line, sometimes from `(dynamic, ct)`. I didn't really find out what the differences is from skimming the specs, did you?

Here, it's tree

File: signatures/FALCON/FALCON/bench/fpemu_1024_Skylake.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+platform: Intel Core i7-6567 (Skylake) 3.3 GHZ
+timings:
+  unit: cycles 
+  gen:  168624000
+  enc|sign: 40404960
```
Comment (dingens) [inline signatures/FALCON/FALCON/bench/fpemu_1024_Skylake.yaml:5]: see above - dynamic

File: signatures/FALCON/FALCON/bench/fpemu_512_Skylake.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+platform: Intel Core i7-6567 (Skylake) 3.3 GHZ
+timings:
+  unit: cycles 
+  gen: 60066000
+  enc|sign: 18799596
```
Comment (dingens) [inline signatures/FALCON/FALCON/bench/fpemu_512_Skylake.yaml:5]: see above - dynamic

File: signatures/FALCON/FALCON/bench/fpnative_1024_Skylake.yaml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+platform: Intel Core i7-6567 (Skylake) 3.3 GHZ
+timings:
+  unit: cycles 
+  gen:  77868000
+  enc|sign: 1626624
```
Comment (dingens) [inline signatures/FALCON/FALCON/bench/fpnative_1024_Skylake.yaml:5]: see above - tree

File: signatures/FALCON/FALCON/bench/fpnative_512_Skylake.yaml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+platform: Intel Core i7-6567 (Skylake) 3.3 GHZ
+timings:
+  unit: cycles 
+  gen:  26136000
+  enc|sign: 1368504
```
Comment (dingens) [inline signatures/FALCON/FALCON/bench/fpnative_512_Skylake.yaml:5]: see above - dynamic

File: signatures/FALCON/FALCON/bench/avx2_512_Skylake.yaml [hunk 7]
```diff
@@ -0,0 +1,7 @@
+platform: Intel Core i7-6567 (Skylake) 3.3 GHZ
```
Comment (dingens) [inline signatures/FALCON/FALCON/bench/avx2_512_Skylake.yaml:1]: Seems `avx2_1024_skylake` is missing

File: signatures/FALCON/FALCON/param/512.yaml [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+security level:
+    classical: 114
+    quantum: 103
+    nist category: 1
+failure probability: 0
+number of operations: inf
+sizes:
+    sk: 1281
+    pk: 897
+    ct|sig: 617
```
[the remark this answers is not in the document]
Comment (Soundes-M) [reply signatures/FALCON/FALCON/param/512.yaml:11]: I think the average 657. Maybe we can add a comment on this.

File: signatures/FALCON/FALCON/bench/avx2_512_Skylake.yaml (hunk 2 above)
Comment (Soundes-M) [reply signatures/FALCON/FALCON/bench/avx2_512_Skylake.yaml:5]: I also did not find a detailed explanation of "dynamic". I ,however, changed all the results to (tree, ct) to be consistent.

**Comment by Soundes-M** [comment]
> Looks nice, thank you!
> 
> I've fixed two things myself (14c0594), what do you think about them?
> 
> * include the impl paper as a source
> * use the expanded secret key size everywhere as I suggested above (for coherency, because the timings were taken using the expanded key). The other option would have been to add the expansion timings to the signing time, but that would increase signing time very significantly, so I thought the large private keys are probably the better choice in most cases.
> * and I clarified the comments (I had the impression that they were not well understandable by people who haven't read the paper, I hope that now it is.
> 
> If you agree with my changes, we can merge it (probably someone else has to actually do it because of our merging rules)

Yes we can merge now.

**Pull request merged**